Gemfile

```diff
@@ -1,3 +1,11 @@
 source 'https://rubygems.org'
 
 gemspec
+
+group :development do
```

Comment:
if you want to move this hear, that's fine, but you'll want to update the docs, because `bundle install` will no longer actually get you ready to run lints and do developments

Reply:
Per the bundler docs:
> By default, bundle install will install all gems in all groups in your Gemfile(5), except those declared for a different platform
so I think its ok to leave as it